src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -3156,6 +3153,15 @@ private bool ScanXmlToken(ref TokenInfo info)
             return info.Kind != SyntaxKind.None;
         }
 
+        private TokenInfo ScanXmlTextLiteralNewLineToken(ref TokenInfo info)
```

Comment:
Why pass by ref and return? I would expect one or the other.

Reply:
No good reason.  Fixed.